raddb/policy.d/canonicalization

```diff
@@ -111,3 +111,16 @@ rewrite_calling_station_id {
                 noop
         }
 }
+
+set_calling_station_id {
+        if (&User-Name && (&User-Name =~ /^${policy.mac-addr-regexp}$/i)) {
```

Comment:
I would be careful here.
policy.mac-addr-regexp would also match "aafacceebb01"  which could be a user.
You could overwrite a valid username.